src/main.rs

```diff
@@ -366,6 +370,7 @@ async fn install_version(args: &Args, program_path: &Path, release: ReleaseInfo)
     );
 
     let archive = get_zip(&release).await?;
+    remove_file(args, program_path)?;
```

Comment:
Maybe, it might be better to introduce another argument to clean up the downloaded archive. `remove_file()` failing could be caused by something temporary (e.g., `terraform` is being used), so I think it would make more sense for this to be user-controlled.